### PR TITLE
Update package.json

### PR DESCRIPTION
**Pull request by Pszek**
React to 15

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -22,7 +22,7 @@
   ],
   "license": "MIT",
   "peerDependencies": {
-    "react": ">=0.13.0 <0.15.0"
+    "react": "^0.14 || ^15"
```
Comment (lencioni) [inline package.json]: Is there any reason this can't still support 0.13.x?

Comment (Pszek) [inline package.json]: Your're right.

**Comment by jheander** [comment]
Solves #11

**Comment by lencioni** [comment]
👍

**Comment by prencher** [comment]
Any change we can get this in?

**Comment by poenneby** [comment]
👍

**Comment by trotzig** [comment]
So sorry for leaving you waiting this long. I haven't kept this repo under my radar for a while. I've made sure I get notifications for all activity now. 

I'll release a new version promptly.

**Comment by trotzig** [comment]
Released in 0.4.0.

**Pull request merged**